test/integration/refund/refund_test.go

```diff
@@ -34,18 +32,19 @@ func TestRefund(t *testing.T) {
 			Capture:      false,
 		}
 
-		payment, err := paymentClient.Create(context.Background(), paymentRequest)
-		if payment == nil {
+		clientPayment := payment.NewClient(cfg)
```

Comment:
o client de refund foi escrito da seguinte forma: "refundClient". Seguir o padrão pra payment
```suggestion
		paymentClient := payment.NewClient(cfg)
```